src/app/movies/movies.component.ts

```diff
@@ -35,8 +36,8 @@ export class MoviesComponent implements OnInit, OnDestroy {
   //   console.log('działa');
   // }
   addMovie(): void {
-    this.movies.unshift(this.movie);
-    this.movie = { id: '', type: 'youtube | vimeo' };
+    // this.movies.unshift(this.movie);
+    this.movie = [];
     console.log(this.movie);
   }
```

Comment:
usun stąd wszystko :p mvies, movie, metody, zostaw pusty komponent. Przygotuj tylko input i jakiś guzik "add" i zrób tak że jak coś wpiszesz w input i klikniesz add, to odpali się metoda addMovie i zrobi console.log tego co jest w inpucie. Jak to zrobisz to zmergujemy kawałek i powiem Ci co dalej